packages/user/src/components/UsersTable/AllUsersTable.tsx

```diff
@@ -0,0 +1,253 @@
+      showFilterMenu: false,
+      showClearButton: false,
+    },
+    {
+      field: "app",
+      header: t("user:invitations.table.defaultColumns.app"),
+      hidden: !visibleColumns.includes("app"),
+      body: (data: { appId: any }) => {
+        return <span>{data.appId || "-"} </span>;
+      },
+      align: "center",
+    },
+    {
+      field: "roles",
+      header: t("table.defaultColumns.roles"),
+      hidden: !visibleColumns.includes("roles"),
+      body: (data) => {
+        return (
+          <>
+            {data.roles.map((role: string, index: number) => (
+              <Tag
+                key={role + index}
+                value={role}
+                style={{
+                  background: role === "ADMIN" ? "#6366F1" : "#22C55E",
+                  width: "5rem",
+                }}
+              />
+            ))}
+          </>
+        );
+      },
+      align: "center",
+    },
+    {
+      field: "status",
+      header: t("table.defaultColumns.status"),
+      hidden: !visibleColumns.includes("status"),
+      body: (data) => {
+        return (
+          <>
+            <Tag
+              value={
+                data.isActiveUser ? t("status.active") : t("status.invited")
+              }
+              style={{
+                background: data.isActiveUser ? "#6366F1" : "#22C55E",
+                width: "5rem",
+              }}
+            />
+          </>
+        );
+      },
+      align: "center",
+    },
+    ...extraColumns,
+    {
+      field: "invitedBy",
+      header: t("user:invitations.table.defaultColumns.invitedBy"),
+      hidden: !visibleColumns.includes("invitedBy"),
+      body: (data) => {
+        if (data.isActiveUser) {
+          return <code>&#8212;</code>;
+        }
+
+        if (data.invitedBy.givenName || data.invitedBy.surname) {
+          return `${data.invitedBy.givenName || ""} ${
+            data.invitedBy.surname || ""
+          }`;
+        }
+
+        return data.invitedBy.email;
+      },
+      align: "center",
+    },
+    {
+      field: "signedUpAt",
+      header: t("table.defaultColumns.signedUpOn"),
+      hidden: !visibleColumns.includes("signedUpAt"),
+      body: (data) => {
+        const date = new Date(data.signedUpAt);
+
+        return date.toLocaleDateString("en-GB");
+      },
+    },
+    {
+      field: "actions",
+      header: t("user:invitations.table.defaultColumns.actions"),
+      hidden: !visibleColumns.includes("actions"),
+      body: (data) => {
+        return (
+          <>
+            {data.isActiveUser ? null : (
+              <InvitationActions
+                onInvitationResent={onInvitationResent}
+                onInvitationRevoked={onInvitationRevoked}
+                invitation={data}
+              />
+            )}
+          </>
+        );
+      },
+      align: "center",
+    },
+  ];
+
+  const rowClassNameCallback = (data: any) => {
+    return `user-${data.id}`;
```

Comment:
have common class name for all rows and then additional classname for user and invitation so that they can be used to style rows differently